packages/cli-types/src/ora.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * The types exported from here are all from the Ora package.
+ * Alternatively we could make this pacakge depend on the Ora
+ * pacakge just to export the actual types.
```

Comment:
Wouldn't re-exporting from a real package be easier from the maintenance perspective? I feel like we depend on Ora anyway somewhere, so no added overhead here.